src/extended/items.rs

```diff
@@ -66,28 +66,37 @@ impl ExtendedHandler for OwnershipValidator {
                 if !last {
                     return;
                 }
-                let name = get_cell_name(cell);
                 let mut owned = 0;
                 let mut unowned = 0;
+                let name = cell.editor_id();
                 for reference in cell.references.values() {
                     let lower = reference.id.to_ascii_lowercase();
                     let scale = reference.scale.unwrap_or(1.);
                     if scale != 1. && self.items.contains(&lower) {
                         println!(
                             "Cell {} contains {} with scale {}",
-                            name, reference.id, scale
+                            cell.editor_id(),
+                            reference.id,
+                            scale
                         );
                     }
-                    if (reference.lock_level.is_some() || !is_empty(&reference.trap))
+
+                    let locked = reference.lock_level.is_some();
+                    let has_trap = matches!(&reference.trap, Some(s) if !s.is_empty());
```

Comment:
Unfortunately they mean different things.

If `trap` is `None` then it would mean it is unspecified, and would fall back to what ever value the previous mod that defined this reference specified. If it's `Some("")` that would mean that it is specifically *removing* the trap that the previous mod specified.

That distinction has to be preserved, so I can't remove the `Option`, but I could add some convenience method that has the semantics of "is it trapped as far as this plugin is aware".